crates/sqlexec/src/planner/logical_plan/create_schema.rs

```diff
@@ -0,0 +1,82 @@
+use super::*;
+
+#[derive(Clone, Debug, Hash, PartialEq, Eq)]
+pub struct CreateSchema {
+    pub schema_name: OwnedSchemaReference,
+    pub if_not_exists: bool,
+}
+
+impl TryFrom<protogen::sqlexec::logical_plan::CreateSchema> for CreateSchema {
+    type Error = ProtoConvError;
+
+    fn try_from(proto: protogen::sqlexec::logical_plan::CreateSchema) -> Result<Self, Self::Error> {
+        let schema_name = proto.schema_name.unwrap().try_into().unwrap();
```

Comment:
Should we be `?`ing these instead of unwrapping?

Reply:
yes, looks like i missed a couple after testing things.